<launch>
    <node pkg="icreate_navigation" type="multifloor_navigation" name="multifloor_navigation" output="screen">
        <param name="move_base_topic" value="/move_base" type="string" />
        <!--<param name="lift_file_path" value="/waypoint/build4_f20lift.csv" type="string" />-->
        <param name="base_frame_id" value="map" type="string" />
        <param name="robot_frame_id" value="base_footprint" type="string" />
        <!--<param name="package_name" value="icreate_navigation" type="string" />-->
    </node>

    <include file="$(find icreate_navigation)/launch/modules/icreate_navigation_stack.launch" />

    <include file="$(find icreate_navigation)/launch/modules/map_server.launch" />

</launch>